src/System.Reflection.Emit/tests/ConstructorBuilder/ConstructorBuilderDefineParameter.cs

```diff
@@ -157,15 +157,16 @@ public void PosTest4()
         }
 
         [Fact]
-        public void PosTest5()
+        public void TestParameterWithZeroSequence()
         {
             CreateConstructorBuilder("PosTest5_Type1", new Type[] { }).DefineParameter(0, ParameterAttributes.None, "p");
```

Comment:
It's just a random string, whether xyz or some * to test locally.. I think it doesn't matter.